### ISSUE TITLE
OpenGL.error.NullFunctionError: Attempt to call an undefined function glutInit, check for bool(glutInit) before calling

### ISSUE THREAD

**Issue by unholyeyebrows** [flathub/com.jetbrains.PyCharm-Community#60]
Hi

I am playing around with PyOpenGL in PyCharm Community edition installed via Flatpak (my host is Fedora 30 Gnome). Running some sample code in PyCharm results in the following error:

`OpenGL.error.NullFunctionError: Attempt to call an undefined function glutInit, check for bool(glutInit) before calling`

However when running the same code from the host command line or via PyCharm installed from the tarball the code works fine.

Steps to duplicate:

1. Create a new project in PyCharm Community edition 2019.1 Flatpak. Create a new venv.
2. Install pygame and pyopengl via the following commands in the project terminal (i.e. in PyCharm itself): python3 -m pip install -U pyopengl
3. Create cube.py using the following code:

```
import sys

try:
  from OpenGL.GLUT import *
  from OpenGL.GL import *
  from OpenGL.GLU import *
except:
  print ('''
ERROR: PyOpenGL not installed properly.  
        ''')

def init():
   glClearColor (0.0, 0.0, 0.0, 0.0)
   glShadeModel (GL_FLAT)

def display():
   glClear (GL_COLOR_BUFFER_BIT)
   glColor3f (1.0, 1.0, 1.0)
   glLoadIdentity ()             # clear the matrix

   gluLookAt (0.0, 0.0, 5.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0)
   glScalef (1.0, 2.0, 1.0)      # modeling transformation
   glutWireCube (1.0)
   glFlush ()

def reshape (w, h):
   glViewport (0, 0, w, h)
   glMatrixMode (GL_PROJECTION)
   glLoadIdentity ()
   glFrustum (-1.0, 1.0, -1.0, 1.0, 1.5, 20.0)
   glMatrixMode (GL_MODELVIEW)

def keyboard(key, x, y):
   if key == chr(27):
      import sys
      sys.exit(0)

glutInit(sys.argv)
glutInitDisplayMode (GLUT_SINGLE | GLUT_RGB)
glutInitWindowSize (500, 500)
glutInitWindowPosition (100, 100)
glutCreateWindow ('cube')
init ()
glutDisplayFunc(display)
glutReshapeFunc(reshape)
glutKeyboardFunc(keyboard)
glutMainLoop()
```

4. Configure the run environment in PyCharm as follows:
5. Script Path: [path to cube.py]
6. Python Interpreter: Project Default (Python 3.7 in the venv I created earlier)
7. Working Directory: [folder containing cube.py] 

The full error received when I attempt to run the code is as follows:

```
/home/.../PycharmProjects/PyOpenGL_cube/venv/bin/python /home/nharvey/PycharmProjects/PyOpenGL_cube/cube.py
Traceback (most recent call last):
  File "/home/.../PycharmProjects/PyOpenGL_cube/cube.py", line 39, in <module>
    glutInit(sys.argv)
  File "/home/.../PycharmProjects/PyOpenGL_cube/venv/lib/python3.7/site-packages/OpenGL/GLUT/special.py", line 333, in glutInit
    _base_glutInit( ctypes.byref(count), holder )
  File "/home/.../PycharmProjects/PyOpenGL_cube/venv/lib/python3.7/site-packages/OpenGL/platform/baseplatform.py", line 407, in __call__
    self.__name__, self.__name__,
OpenGL.error.NullFunctionError: Attempt to call an undefined function glutInit, check for bool(glutInit) before calling

Process finished with exit code 1
```

**Comment by unholyeyebrows** [comment]
I have just tried the build 3898 and confirm this fixes the issue. Fantastic turn-around - many thanks again.

**Comment by avdodger** [comment]
Posted this on another page too, but here because I need answers ASAP.  it's August 2020 and this problem exists. I've downloaded pyOpenGL and/or freeGlut to no avail. I continue to get fails for using an undefined function glutInit()... I've got .dlls in the directory where my code is, and other places in my path - they're all avoided. I'm running either Python 3.8 using Idle or Anaconda with Spyder - nothing works. Any ideas? And yes, I'm only a couple months into Python.
